Shared/Cloudinary.cs

```diff
@@ -1081,38 +1081,45 @@ private string GetUploadUrl(string resourceType)
         /// <summary>
         /// Async call to get a list of folders in the root asynchronously.
         /// </summary>
+        /// <param name="parameters">(optional) Parameters for managing folders list.</param>
         /// <returns>Parsed result of folders listing.</returns>
-        public Task<GetFoldersResult> RootFoldersAsync()
+        public Task<GetFoldersResult> RootFoldersAsync(GetFoldersParams parameters = null)
         {
             return m_api.CallApiAsync<GetFoldersResult>(
                 HttpMethod.GET,
-                GetFolderUrl(),
+                GetFolderUrl(parameters: parameters),
                 null,
                 null);
         }
 
         /// <summary>
         /// Gets a list of folders in the root.
         /// </summary>
+        /// <param name="parameters">(optional) Parameters for managing folders list.</param>
         /// <returns>Parsed result of folders listing.</returns>
-        public GetFoldersResult RootFolders()
+        public GetFoldersResult RootFolders(GetFoldersParams parameters = null)
         {
-            return m_api.CallApi<GetFoldersResult>(HttpMethod.GET, GetFolderUrl(), null, null);
+            return m_api.CallApi<GetFoldersResult>(
+                HttpMethod.GET,
+                GetFolderUrl(parameters: parameters),
+                null,
+                null);
         }
 
         /// <summary>
         /// Gets a list of subfolders in a specified folder asynchronously.
         /// </summary>
         /// <param name="folder">The folder name.</param>
         /// <param name="cancellationToken">(Optional) Cancellation token.</param>
+        /// <param name="parameters">(Optional) Parameters for managing folders list.</param>
         /// <returns>Parsed result of folders listing.</returns>
-        public Task<GetFoldersResult> SubFoldersAsync(string folder, CancellationToken? cancellationToken = null)
+        public Task<GetFoldersResult> SubFoldersAsync(string folder, CancellationToken? cancellationToken = null, GetFoldersParams parameters = null)
```

Comment:
Parameters order is inconsistent with other methods `(subject, parameters, cancellation token)`
Please fix the order and add and overloaded method ` public Task<GetFoldersResult> SubFoldersAsync(string folder, CancellationToken? cancellationToken = null)` for backwards compatibility.